arcgis-ios-sdk-samples/Edit data/Update attributes (feature service)/EditAttributes.storyboard

```diff
@@ -40,74 +40,59 @@
-                                </connections>
-                            </tableView>
-                            <navigationBar contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ciP-GV-Hz3">
-                                <rect key="frame" x="0.0" y="20" width="375" height="44"/>
-                                <items>
-                                    <navigationItem title="Select new damage type" id="BI7-Jv-qUJ">
-                                        <barButtonItem key="rightBarButtonItem" systemItem="cancel" id="wkb-gu-hjX">
-                                            <connections>
-                                                <action selector="cancelAction" destination="J6E-XC-43c" id="974-1C-gwz"/>
-                                            </connections>
-                                        </barButtonItem>
-                                    </navigationItem>
-                                </items>
-                                <connections>
-                                    <outlet property="delegate" destination="J6E-XC-43c" id="Wbw-X0-C47"/>
-                                </connections>
-                            </navigationBar>
-                        </subviews>
-                        <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
-                        <constraints>
-                            <constraint firstAttribute="trailing" secondItem="Y6b-fZ-TDx" secondAttribute="trailing" id="8JR-99-2OF"/>
-                            <constraint firstItem="Y6b-fZ-TDx" firstAttribute="leading" secondItem="IeL-vL-NqC" secondAttribute="leading" id="BYc-er-Eko"/>
-                            <constraint firstItem="Y6b-fZ-TDx" firstAttribute="top" secondItem="ciP-GV-Hz3" secondAttribute="bottom" id="V38-tW-lXB"/>
-                            <constraint firstItem="ciP-GV-Hz3" firstAttribute="leading" secondItem="IeL-vL-NqC" secondAttribute="leading" id="Xoa-mC-s0b"/>
-                            <constraint firstAttribute="trailing" secondItem="ciP-GV-Hz3" secondAttribute="trailing" id="feG-J4-by0"/>
-                            <constraint firstItem="ciP-GV-Hz3" firstAttribute="top" secondItem="llt-tw-lsS" secondAttribute="top" id="kFH-9i-ATq"/>
-                            <constraint firstAttribute="bottom" secondItem="Y6b-fZ-TDx" secondAttribute="bottom" id="lDS-mN-Ugi"/>
-                        </constraints>
-                        <viewLayoutGuide key="safeArea" id="llt-tw-lsS"/>
-                    </view>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <prototypes>
+                            <tableViewCell contentMode="scaleToFill" selectionStyle="default" indentationWidth="10" reuseIdentifier="EAOptionsCell" id="ltd-qf-aNW">
```

Comment:
The cell's style really should have been `basic` (`style="IBUITableViewCellStyleDefault"`) rather than custom. Could you please fix that?